storage/client/queue/src/main/java/com/azure/storage/common/credentials/SharedKeyCredential.java

```diff
@@ -116,10 +118,10 @@ public String computeHmac256(final String stringToSign) {
             byte[] utf8Bytes = stringToSign.getBytes(StandardCharsets.UTF_8);
             return Base64.getEncoder().encodeToString(hmacSha256.doFinal(utf8Bytes));
         } catch (final NoSuchAlgorithmException e) {
-            LOGGER.asError().log(e.getMessage());
+            logger.asError().log(e.getMessage());
             throw new RuntimeException(e);
```

Comment:
These should all use the `logger.logAndThrow` API.

Reply:
We are depending on preview 2 for Azure core. I have no visibility for preview 3 API.

I have an issue for this after move to the azure core preview 3:
 https://github.com/Azure/azure-sdk-for-java/issues/4551

Reply:
Can't we upgrade to preview 3? Otherwise we will end up with queues being incompatible with all other Java client libraries in the next preview round.

Reply:
Blobs, Files, and Queues will be moved to Azure Core Preview 3 once #4489 merges in